ReactWindows/ReactNative.Shared/UIManager/AccessibilityTrait.cs

```diff
@@ -0,0 +1,64 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// NOTE: This enum is organized based on priority of these traits (0 is the lowest),
+    /// which can be assigned to an accessible object. On native, all traits are combined as
+    /// a list. On desktop, trait with the maximum value is picked. Whenever you are adding
```

Comment:
>desktop [](start = 19, length = 7)

We mention "desktop" here and "native" in the previous sentence. I'm confused by the distinction. Maybe this was copied from ReactXP and it walk talking about electron vs React Native?

Reply:
agree. removed.

Reply:
removed the comment due to refactoring of the enum.

---
In reply to: [158560642](https://github.com/reseul/react-native-windows/pull/1#discussion_r158560642) [](ancestors = 158560642)